### PR TITLE
chore(deps): Replace colors with chalk

### PR DESCRIPTION
**Pull request by nwalters512**
This PR replaces [`colors`](https://www.npmjs.com/package/colors), which was [recently vandalized by its maintainer](https://blog.sonatype.com/npm-libraries-colors-and-faker-sabotaged-in-protest-by-their-maintainer-what-to-do-now), with [`chalk`](https://www.npmjs.com/package/chalk), which is what we use everywhere else we need to color text for the terminal.

This will unblock https://github.com/PrairieLearnInc/PrairieTest/pull/453.

### CODE REVIEW

File: lib/databaseDescribe.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
+// @ts-check
```
Comment (mwest1066) [inline lib/databaseDescribe.js:1]: I'm looking forward to this being real TS!

File: tools/pg_describe.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
```
Comment (mwest1066) [inline tools/pg_describe.js:1]: Any idea why these files don't show up as rename in GitHub? I assume the changes in the files were minor?

**Review by mwest1066 — APPROVED** [review]

File: tools/pg_describe.js (hunk 2 above)
Comment (nwalters512) [reply tools/pg_describe.js:1]: "minor" in the sense that Prettier was run on them for the first time ever 😛 if it helps make the history cleaner (or at least easier to `git blame`), I can save the rename for another PR.

Comment (nwalters512) [reply tools/pg_describe.js]: I did this in [c802148](https://github.com/PrairieLearn/PrairieLearn/pull/5799/commits/c8021484ae941fdd9aac538f5f753a49af2f6305).

Comment (nwalters512) [reply tools/pg_describe.js]: Actually, changed my mind, I'm splitting this up into a few more PRs to keep history clean:
#5800 and #5801.

Comment (mwest1066) [reply tools/pg_describe.js]: Nice, thanks, I was struggling to see what was going on but now it's crystal clear.

Comment (nwalters512) [reply tools/pg_describe.js]: The overall diff for this PR should be much clearer now!

**Pull request merged**